src/server/system_services/system_server.js

```diff
@@ -142,6 +142,10 @@ function new_system_defaults(name, owner_account_id) {
         _id: system_store.new_system_store_id(),
         name: name,
         owner: owner_account_id,
+        state: {
```

Comment:
why do we need it ?

Reply:
I am not sure what do you mean. This field holds the status of the system

Reply:
the timestamp